.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+If you are creating an issue to seek help or support with your
+_usage_ of Scrapy, however, please consider posting instead
+to a more appropriate forum: We support Reddit, IRC, and Stack
+Overflow. Check our Community portal for details:
+https://scrapy.org/community/
```

Comment:
I wonder if we should link to the SUPPORT.md document here, to be a bit more consistent. There are too many pages to keep in sync already:

* new SUPPORT.md document;
* https://scrapy.org/community/
* https://doc.scrapy.org/en/latest/#getting-help
* https://doc.scrapy.org/en/latest/contributing.html#reporting-bugs